code/src/index.css

```diff
@@ -1,13 +1,203 @@
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  background: #181818;
+  color: #fff;
+  font-family: -apple-system, BlinkMacSystemFont, 'Helvetica', 'Segoe UI',
+    'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
+    'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+a {
+  color: #fff;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #fff;
+  border-bottom: 2px solid #fff;
+}
+
+/* ----- Main wrapper and heading ----- */
+.main-wrapper {
+  padding: 30px;
+}
+
+.main--heading {
+  padding: 0 0 15px 0;
+  border-bottom: #212121 1px solid;
+  margin-bottom: 20px;
+  font-size: 18px;
+  font-style: bold;
+}
+
+.all-content--wrapper {
+  display: flex;
+}
+
+/* ----- Album-section ----- */
+.all-albums--wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.album--wrapper {
+  width: calc(25% - 10px);
+  margin-bottom: 20px;
+}
+
+.image--wrapper {
+  position: relative;
+}
+
+/* Icon styling */
+.icons--wrapper {
+  visibility: hidden;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: rgba(33, 33, 33, 0.4);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.image--wrapper:hover .icons--wrapper {
+  visibility: visible;
+}
+
+.icon--wrapper {
+  width: 30%;
+  height: auto;
+}
+
+.icon--wrapper:first-child,
+.icon--wrapper:last-child {
+  width: 20%;
+}
+
+.play {
+  width: 75%;
+  height: auto;
+  opacity: 0.8;
+}
+
+.play:hover {
+  width: 79%;
+  opacity: 1;
+}
+
+.heart,
+.dots {
+  width: 40%;
+}
+
+.icon {
+  cursor: pointer;
+  filter: invert(1);
+}
+
+/* Image styling */
+.album--image {
+  width: 100%;
+  height: auto;
+}
+
+.image-link:hover {
+  border: none;
+}
+
+/* Album & artist name styling */
+.album-name,
+.artist-name {
+  font-size: 14px;
+  font-size: 600;
+  line-height: 1.4;
+}
+
+.album-name {
+  margin-top: 10px;
+}
+
+.artist-name {
+  color: #a0a0a0;
+  margin: 5px 0 30px 0;
+}
+
+.artist-text {
+  color: #a0a0a0;
+}
+
+/* Playlist */
+.logo {
+  width: 10%;
+  height: auto;
+  filter: invert(1);
+  margin-bottom: 20px;
+}
+
+.logo:hover {
+  cursor: pointer;
+}
+
+.all-playlists--wrapper {
+  min-width: 245px;
+}
+
+.list--title {
+  text-transform: uppercase;
+  font-size: 12px;
+  line-height: 2;
+  letter-spacing: 1.5px;
+}
+
+.list--wrapper {
+  display: flex;
+  flex-direction: column;
+  line-height: 2;
+  list-style-type: none;
+}
+.list--link {
+  color: #a0a0a0;
+}
+
+.list--link:hover {
+  color: #fff;
+  border: none;
+  cursor: default;
```

Comment:
`cursor` property should be added to default element (by default I mean not hovered, so rather on `.link--link` than on `.list--link:hover`).
Moreover, UX standards says that if element is clickable, then cursor should be set to `pointer`